core/common/src/main/java/alluxio/grpc/GrpcServer.java

```diff
@@ -39,12 +48,14 @@
    *
    * @param server the wrapped server
    * @param authServer the authentication server
+   * @param closer resources to close before shutting down server
```

Comment:
who owns (is responsible for closing) the closer?

Reply:
The `GrpcServer`

Reply:
Can you add a comment to explicitly mention who closes this closer?

Reply:
Changed it to ` * @param closer resources to close during shutting down of this server`